src/Algebra/Graph/NonEmpty/AdjacencyMap/Internal.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
I'm not sure, but it seems to me that here we derive `Generic` using `GeneralisedNewtypeDeriving`, which is enabled by default. Could you please check?

Reply:
FWIW here's what I get without `DeriveGeneric`:

```
src/Algebra/Graph/AdjacencyMap/Internal.hs:136:51-57: error:
    • Can't make a derived instance of ‘Generic (AdjacencyMap a)’
        (even with cunning GeneralizedNewtypeDeriving):
        You need DeriveGeneric to derive an instance for this class
    • In the newtype declaration for ‘AdjacencyMap’
    |
136 |     adjacencyMap :: Map a (Set a) } deriving (Eq, Generic)
    |                                                   ^^^^^^^

```

Reply:
Aha, I see, thanks for checking!